README.md

```diff
@@ -16,26 +30,62 @@ meteor add johanbrook:publication-collector
 
 ## Usage
 
+This package is server-only and can't be imported on the client.
+
 ```js
-// In a typical BDD style test suite:
+// server/myPublication.test.js
+
+import { PublicationCollector } from 'meteor/johanbrook:publication-collector';
 
-describe('Publication', function() {
+describe('myPublication', function() {
   it('should publish 10 documents', function(done) {
-    // Pass user context in constructor.
-    const collector = new PublicationCollector({userId: Random.id()});
+    const collector = new PublicationCollector({ userId: Random.id() });
 
-    // Collect documents from a subscription with 'collect(name, [arguments...], [callback])'
-    collector.collect('publicationName', 'someArgument', (collections) => {
+    collector.collect('myPublication', firstPublicationArg, secondPublicationArg, (collections) => {
       assert.equal(collections.myCollection.length, 10);
       done();
     });
   });
 });
 ```
 
+### PublicationCollector
+
+```js
+const collector = new PublicationCollector( opts );
+```
+
+`opts` may have the following attributes:
+- `userId`: Add a `this.userId` to the publication's context
+- `delayInMs`: By default, `collect` callbacks are called when the publication is ready. If you use this option, the callbacks will be called `delayInMs` milliseconds after the publication is ready.
```

Comment:
Gotcha. Hm, it feels a tad hacky with a timeout though. No way we can solve it more elegantly?